app/components/shortcut-editor.jsx

```diff
@@ -0,0 +1,130 @@
+import React from 'react';
+import AutoSave from './auto-save';
+import handleInputChange from '../lib/handle-input-change';
+import Shortcut from '../classifier/tasks/shortcut';
+
+export default class ShortcutEditor extends React.Component {
+  constructor(props) {
+    super(props);
+    this.addAnswer = this.addAnswer.bind(this);
+  }
+
+  toggleShortcut(e) {
+    let nextTaskID;
+
+    if (e.target.checked) {
+      const taskCount = Object.keys(this.props.workflow.tasks).length;
```

Comment:
`Uncaught TypeError: Cannot read property 'props' of undefined` when trying to use the checkbox button on a test project.